packages/shared/src/stateChainGateway/__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -4,13 +4,7 @@ exports[`executeRedemption executes the redemption 1`] = `
 [
   "0x1234",
   {
-    "network": "sisyphos",
     "nonce": 1,
-    "signer": VoidSigner {
```

Comment:
this was the initial reason for the pr - we pass the `signer` and `network` to `ethers.js` at the moment. this causes ethers to throw an error that it does not expect a `signer` property